scripts/env/pull-env.sh

```
@@ -141,14 +145,53 @@ pull_all_services() {
+
 generate_public_env() {
-  ./scripts/env/generate-public-env.sh $1
+  bash ./scripts/env/generate-public-env.sh $1
 }
 
 export -f pull_env_file
 export INFISICAL_TOKEN sFlag PLATFORM
 
 case "$TARGET_ENV" in
+  "act")
+    printf "${INFO}Pulling act secrets...${NC}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)